src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -743,7 +743,7 @@ private function getProductURL(
             $ean13,
             $language->id,
             null,
-            !$canonical && $product['id_product_attribute'] > 0 ? $product['id_product_attribute'] : null,
+            !$canonical && $product['id_product_attribute'] > 0 ? $product['id_product_attribute'] : $product['cache_default_attribute'] > 0 ? $product['cache_default_attribute'] : null,
```

Comment:
It's unreadable, could you use variable instead?

Reply:
"Nested ternary operators without explicit parentheses" is a deprecated feature for PHP 7.4: https://www.php.net/manual/en/migration74.deprecated.php#migration74.deprecated.core.nested-ternary